pullrequest.rst

```diff
@@ -166,6 +166,11 @@ any discussion of what the pull request is trying to solve (e.g. fixing a
 spelling mistake), then the pull request needs to have the "trivial" label
 added to it.
 
+If your pull request involves several commits as a result of addressing code
+review comments, please do not squash them.  When you squash your commits,
+reviewers lose the ability to view the diff of one commit to the next
+and are therefore unable to easily verify whether their comments have been
+addressed.
```

Comment:
At the end, should it provide the describe that the member who contribute with PR, when approved the PR, he/she will finish with the squash of the commit.